### DISCUSSION TITLE
Can I use this fork for AndroidTV as well?

### DISCUSSION THREAD

**Discussion by jrhager84** [react-native-tvos/react-native-tvos#374] [Q&A] [answered]
I'm trying to keep the code duplication as low as possible, and reading through the documentation states that this is a direct fork of React Native core, with the exception that it includes only the necessary changes for tvOS to work. In theory, doesn't that mean that Android would *also* work in this fork as well? Thanks!

**Comment by douglowder** [comment, marked as answer]
I'll have to make that documentation more clear... this repo supports both Apple TV and Android TV!

**Reply by jrhager84** [reply]
I figured as much, but wanted to be sure. Thank you for your prompt response! 😊
